base/scripts/build_tool.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+#
+# build    ... builds a local version
+# run      ... runs the locally built version
+# run-prod ... downloads the version from docker registry and runs it
+# publish  ... publishes the locally built version
+
+set -o errexit; set -o pipefail; set -o nounset
+
+readonly RE_DOCKER_REGISTRY
+readonly RE_DOCKER_PASSWORD
+readonly RE_DOCKER_USERNAME
+
+readonly _BT_MICROSERVICE_NAME="pb-base"
+
+declare _BT_VCS_REF; _BT_VCS_REF="v1.0"
+# $(git rev-parse --short HEAD)
+readonly _BT_MICROSERVICE_VERSION=${_BT_VCS_REF}
```

Comment:
Set the version here, if you want the image tag to be the version string instead of the VCS reference, which makes sense for base images.